packages/account/src/Components/poi/idv-document-submit/idv-document-submit.tsx

```diff
@@ -205,7 +206,7 @@ const IdvDocumentSubmit = observer(
                                     />
                                 </div>
                             )}
-                            <FormBody className='form-body' scroll_offset={is_mobile ? '180px' : '80px'}>
+                            <FormBody className='form-body' scroll_offset={!isDesktop ? '180px' : '80px'}>
```

Comment:
For better code readability.
```suggestion
                            <FormBody className='form-body' scroll_offset={isDesktop ? '80px' : '180px'}>
```

Reply:
its fixed 👍 👍